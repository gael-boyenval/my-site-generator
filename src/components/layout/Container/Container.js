import styled from 'styled-components';
import { unit, spaces } from 'Styles/tokens';
import { rem } from 'Styles/utils';

const Container = styled.div`
  max-width: ${rem(unit * 108)};
  margin: 0 auto;
  padding: 0 ${rem(spaces.xl)};
`;

export default Container;
