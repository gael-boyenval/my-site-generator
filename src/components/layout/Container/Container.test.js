/* eslint-disable no-undef */
import React from 'react';

import Container from './Container';

describe('Component Container', () => {
  it('should be render without crash', () => {
    const component = shallow(<Container />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
