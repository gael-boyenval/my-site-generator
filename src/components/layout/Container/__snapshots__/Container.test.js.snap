// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Container should be render without crash 1`] = `
.c0 {
  width: 100%;
  max-width: 129.6rem;
  margin: 0 auto;
  padding: 0 2.4rem;
}

<div
  className="c0"
/>
`;
